en/Plugins/Core plugins.md

```diff
@@ -29,5 +29,4 @@ Plugins can be managed through the "Settings" button on the bottom left. Some of
 1. [[Open in default app]]
 1. [[Workspaces]]
 1. [[File recovery]]
-1. [[Publish]]
```

Comment:
maybe leave it as `[[Introduction to Obsidian Publish|Publish]]` to be consistent with Sync?

Reply:
Good catch!